tests/matrix_free/laplace_operator_01.cc

```diff
@@ -108,7 +108,7 @@ void test ()
   }
 
   MatrixFreeOperators::LaplaceOperator<dim,fe_degree,fe_degree+1, 1, number> mf;
-  mf.initialize(mf_data);
+  mf.initialize(std_cxx11::make_shared<MatrixFree<dim,number> >(mf_data));
```

Comment:
It's done except for the `MatrixFree` objects in `MGLevelObject`. I will let you do it when you clean up `step-37`.